src/renderer/components/MainPage/Importers/Importer.vue

```diff
@@ -1,60 +1,41 @@
 <template>
-  <el-card
-    v-if="decryptedImporter"
-    :body-style="{ padding: '0px' }"
-  >
-    <el-collapse-item :name="decryptedImporter.key">
```

Comment:
The `Importer.vue` component shouldn't contain an `el-collapse-item` that depends on `el-collapse` in `Importers.vue`.
Each component should stand alone.